R/utils-fs.R

```diff
@@ -50,6 +50,15 @@ file_copy_to <- function(pkg,
   file_copy(from_paths[!eq], to_paths[!eq], overwrite = overwrite)
 }
 
+# Checks init_site() first.
+create_subdir <- function(dest_base, subdir) {
+  if (!fs::dir_exists(dest_base)) {
+    init_site()
```

Comment:
I forgot to include `pkg` as an argument here. I will look at the implications locally. Hopefully, not too difficult to fix